notebooks/work-in-progress/eia-epacems-crosswalk.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p>I totally agree with you on this, but I'm inclined to keep it in because it doesn't do any harm. What it does is add more EIA ids to the crosswalk if there <em>happens</em> to be an exact name match between the <code>plant_name_eia</code> and the <code>plant_name_epa</code>. In this case there are actually 2883 exact matches so it's pretty useful for filling in the gaps! </p>